state/model.go

```diff
@@ -414,24 +414,16 @@ func (ctlr *Controller) NewModel(args ModelArgs) (_ *Model, _ *State, err error)
 		// the same "owner" and "name" in the collection. If the txn is
 		// aborted, check if it is due to the unique key restriction.
 		name := args.Config.Name()
-		qualifierTerm := bson.DocElem{"owner", owner.Id()}
-		if args.Type == ModelTypeCAAS {
```

Comment:
so for caas, we have owner as well, right?

Reply:
yeah, all models have owners